cms-web-automation-test-master/tests/acceptance/Portal/ContentCest.php

```diff
@@ -275,12 +425,30 @@ public function sortContentByTitle(AcceptanceTester $I)
         $I->waitForText('Y Movie', 30, "//tr[4]/td[" . ContentPage::$titleCol . "]");
         $I->waitForText('X Movie', 30, "//tr[5]/td[" . ContentPage::$titleCol . "]");
     }
+    */
+
+    /**
+     * TESTRAIL TESTCASE ID: C9173
+     *
+     * @group test_priority_2
+     */
+    public function sortContentByTitle(ContentSteps $I)
+    {
+        $I->wantTo('Verify content can be sorted by title. - C9173');
+        $I->amOnContentPage();
+        $I->waitForElementVisible(ContentPage::$table_header);
+        $I->click(ContentPage::$table_header_title);
```

Comment:
Also look to http://codeception.com/docs/07-AdvancedUsage#Examples
if this can help you